docs/rules/prefer-destructuring.md

```diff
@@ -21,6 +21,8 @@ The rule has a second object with a single key, `enforceForRenamedProperties`, w
 - Accessing an object property whose key is an integer will fall under the category `array` destructuring.
 - Accessing an array element through a computed index will fall under the category `object` destructuring.
 
+The `--fix` option on the command line fixes only problems reported in variable declarations, and among them only those that fall under the category `object` destructuring. Furthermore, the name of the declared variable has to be the same as the name used for non-computed member access in the initializer. For example, `var foo = object.foo` can be automatically fixed by this rule. Problems that involve computed member access (e.g., `var foo = object[foo]`) or renamed properties (e.g., `var foo = object.bar`) are not automatically fixed.
```

Comment:
Adding this also covers the `var foo = object['foo'];` case, which we don't fix either and for which we already have a test asserting the same.